Dangerfile

```diff
@@ -40,6 +43,11 @@ files_to_check += ["Cakefile", "fastlane/settings.yml.erb", "fastlane/Fastfile",
   message("`#{file}` modified")
 end
 
+diff = github.pr_diff
+# Ensure we keep using secure https:// references instead of http://
+fail("Detected unsecure `http://` use in `#{file}` - `#{line}`") if diff =~ /\+\s*http:\/\/.*/
```

Comment:
may be we should check file extension as `http://` could be added to README and that would be fine, no?

May be leave like this but use `warn` instead of fail to avoid breaking the PR due to false positives